### ISSUE TITLE
autonag bot nagged about setting severity, for the same bug two days in a row

### ISSUE THREAD

**Issue by dholbert** [mozilla/relman-auto-nag#1368]
Apparently the bot is pretty persistent. :)

The bot posted a nag/needinfo comment to set severity on 3/29:
https://bugzilla.mozilla.org/show_bug.cgi?id=1759667#c7

I cleared the needinfo without setting the severity yet, since we were awaiting information from the reporter which would help us assess severity.

Then the bot posted the same nag/needinfo comment the next day on 3/30 (today).

Maybe it needs some sort of suppression timeout, to avoid rapidly building up comments when the triage owner has some reason to decline-to-take-the-nagged-action right away?

**Comment by dholbert** [comment]
Here's a screenshot showing the exchange:

![image](https://user-images.githubusercontent.com/426803/160916157-cd0adf1e-e040-4ef0-afc0-d0c8394f8e1d.png)

**Comment by dholbert** [comment]
(in this particular case, it's ~fine because we received the information from the reporter and I think I can indeed make a severity assessment now.  But if we were still waiting on the reporter, it could get frustrating to have the bot persistently re-setting needinfo. Maybe the current expectation is that the triage-owner should leave the needinfo open, and then the bot won't re-nag?  If so: that's non-obvious, and it could take a few duplicate-nags and build up a few comments' worth of bot-spam before folks figure that out in individual cases.)

**Comment by marco-c** [comment]
Duplicate of #985.
